### ISSUE TITLE
Titles of radio list headings targeted for screen reader users

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#109]
Currently, headings of columns in the radio lists:
- contain short heading text like "MHz" or "kHZ" in frequency or "Pol." in polarization;
- provide HTML `title` attribute (native browser popup) like "Frequency" or "Polarization" (implemented in #88).

This works for typical graphical users pretty good. Simple popup for columns like "MHz" seems to be good for newbies, as well. Unfortunately, it seems to me that current implementation may be insufficient for not experienced RadioLista users who use screen reader. 

I did some tests with NVDA on Windows 10. When column headings are focused manually by "tab" button both heading content and heading tooltip are spoken. Unfortunately, when the whole page content is spoken automatically by screen reader, only heading content is spoken and heading tooltip is omitted. 

I would like to check how this kind of table is implemented in different a11y-friendly sites. Maybe I can use aria-label or other aria-* attribute to provide alternative text for column headings only for screen readers with both short text (like "MHz") and long text (like "frequency").

#15
